src/pages/reference/handlers/openapi.md

```diff
@@ -100,7 +100,7 @@ We use path-based naming. So names could be structured like `query_getUsers_item
   ]
 }
 ```
-
+<!-- 
 ### From Environmental Variable
 
 `MY_API_TOKEN` is the name of the environmental variable that you have the value for.
```

Comment:
Please remove this too since we dont allow them to access env variable. Instead we can mention about headers from context?

Reply:
the lines you mentioned are commented out, so they won't be visible in the docs, they are just there for my reference if we need them later